backend/app/services/common.js

```diff
@@ -404,3 +405,20 @@ var getFlagsForTask = function* (req, tasks) {
 }
 
 exports.getFlagsForTask = getFlagsForTask;
+
+var insertProjectUser = function* (req, userId, projectId) {
```

Comment:
This is good! There are quite a number of things that we could actually move to the common.js file. I have repeated code in a few places.

Reply:
Cool.